library/unwind/src/unwinding.rs

```diff
@@ -0,0 +1,102 @@
+#![allow(nonstandard_style)]
```

Comment:
Most of these are available from `unwinding::abi`, can this be used instead?

Reply:
The problem is that the callee uses raw pointers, so most of the functions here are casting pointers to references. The calling code could be modified to use references rather than pointers, but I didn't want to touch such fundamental structures.

Reply:
Alright, I've changed the signatures and verified that it still works.